src/styles/components/_cookieBanner.scss

```diff
@@ -0,0 +1,54 @@
+.util-cookie-banner-container {
+  flex-direction: column;
+  border-width: 1px solid;
+  border-color: $black;
+  border-style: solid;
+  background-color: $white;
+  position: fixed;
+  max-width: 480px;
+  min-width: 480px;
+  bottom: 0;
+  left: 0;
+  height: 100%;
+  box-shadow: 0px 2px 10px -3px $x-d-grey;
+  font-size: 16px;
+  z-index: 1001;
+
+  &--show {
+    display: flex;
+    transition: transform 0.4s;
+    transform: translate(0%);
+  }
+
+  &--hide {
```

Comment:
should this be __hide?

Reply:
well, if you think about it there shouldn't be verb in the class name at all. it is either a noun or a adjective.

Reply:
This has now been changed to an adjective.